src/Controller/AppController.php

```diff
@@ -261,36 +261,36 @@ public function beforeRender(Event $event)
             $this->set('_serialize', true);
         }
     }
-
+    
     public function beforeFilter(Event $event)
     {
         $user = $this->Auth->user();
         if(!is_null($user)){
             $userData = TableRegistry::get("security_users")->get($user['id']);
-            try {
```

Comment:
This code block is related to password policy, right? Why is it deleted?

Reply:
yes

Reply:
It's not removed